django_enumfield/tests/test_enum.py

```diff
@@ -32,16 +33,19 @@ def test_enum_field_save(self):
 
         self.assertRaises(InvalidStatusOperationError, setattr, lamp, 'state', 99)
 
-        # Test model with EnumField WITH _transitions
+        # Test model with EnumField WITH __transitions__
         person = Person.objects.create()
         pk = person.pk
         self.assertEqual(person.status, PersonStatus.ALIVE)
         person.status = PersonStatus.DEAD
         person.save()
+        self.assertTrue(isinstance(person.status, PersonStatus))
         self.assertEqual(person.status, PersonStatus.DEAD)
 
         person = Person.objects.get(pk=pk)
         self.assertEqual(person.status, PersonStatus.DEAD)
+        # TODO: find a way to get only one type
+        self.assertTrue(isinstance(person.status, (PersonStatus, int)))
```

Comment:
I was looking into wrong line for a while trying to figure out what was going on and that is still sitting in my head :smiley_cat: Need to take a break